pthread_support.c

```diff
@@ -394,6 +394,7 @@ STATIC pthread_t GC_mark_threads[MAX_MARKERS];
 {
     int i;
     pthread_attr_t attr;
```

Comment:
pthread_sigmask is not available on some targets, see description of GC_NO_PTHREAD_SIGMASK